### DISCUSSION TITLE
How to disable Next.js client side routing(Link) globally

### DISCUSSION THREAD

**Discussion by yassinebridi** [vercel/next.js#30215] [Help] [answered]
Due to this issue https://github.com/vercel/next.js/issues/27783, I can't use the Link component at the moment, even though I updated to this version https://github.com/vercel/next.js/releases/tag/v11.1.3-canary.14, the issue is still standing.

So I would like to momentarily disable client-side routing globally until the issue is fixed, is this possible?

**Comment by jamesmosier** [comment, marked as answer]
There is no way to disable links globally through any config option, but you could potentially use [`beforePopState`](https://nextjs.org/docs/api-reference/next/router#routerbeforepopstate) to listen for the `popstate` event and redirect using `window.location.herf` then return `false` to prevent Next from handling the route natively.

**Reply by yassinebridi** [reply]
This will definitely do, thanks you @jamesmosier
